metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/ProfilerClient.java

```diff
@@ -23,13 +23,14 @@
 import org.apache.metron.profiler.ProfileMeasurement;
 import org.apache.metron.profiler.ProfilePeriod;
 
+import java.io.Closeable;
 import java.util.List;
 import java.util.Optional;
 
 /**
  * An interface for a client capable of retrieving the profile data that has been persisted by the Profiler.
  */
-public interface ProfilerClient {
+public interface ProfilerClient extends Closeable {
```

Comment:
Needs to be closable so that the underlying Hbase connections can be cleaned-up.